sys/include/net/ieee802154/radio.h

```diff
@@ -588,14 +588,14 @@ struct ieee802154_radio_ops {
      *
      * @param[in] dev IEEE802.15.4 device descriptor
      * @param[in] short_addr the IEEE802.15.4 short address
-     * @param[in] ext_addr the IEEE802.15.4 extended address
+     * @param[in] ext_addr the IEEE802.15.4 extended address (Network Endian)
      * @param[in] pan_id the IEEE802.15.4 PAN ID
      *
      * @return 0 on success
      * @return negative errno on error
      */
-    int (*set_hw_addr_filter)(ieee802154_dev_t *dev, uint8_t *short_addr,
-                              uint8_t *ext_addr, uint16_t pan_id);
+    int (*set_hw_addr_filter)(ieee802154_dev_t *dev, network_uint32_t *short_addr,
```

Comment:
Short address is 16 bit :wink:

Reply:
yes, of course, my bad :see_no_evil:

Reply:
https://github.com/RIOT-OS/RIOT/pull/14371/commits/6a2ee4fceced996576eb2ba3e06463a39687a629